.github/workflows/publish-nightly.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "22 0 * * *"
+
+jobs:
+  publish-artifacts:
+    name: Publish artifacts to Apache Nexus
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko-persistence-cassandra'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Enable jvm-opts
+        run: cp .jvmopts-ci .jvmopts
```

Comment:
this repo doesn't seem to have a .jvmopts-ci file

Reply:
good point